docs/commands/touch.md

```diff
@@ -52,4 +52,4 @@ Create three files at oince
   0 │ a                  │ File │      0 B │ 0 sec ago
   1 │ b                  │ File │      0 B │ 0 sec ago
   2 │ c                  │ File │      0 B │ 0 sec ago
-────┴────────────────────┴──────┴──────────┴──────────────
\ No newline at end of file
+────┴────────────────────┴──────┴──────────┴──────────────
```

Comment:
Hmm, this is weird.  This line change does not show up in my diff on GitHub Desktop:

<img width="1271" alt="Screen Shot 2021-01-01 at 5 28 38 PM" src="https://user-images.githubusercontent.com/19867440/103447268-d9359d00-4c56-11eb-901a-4caa376e2c1d.png">